### PR TITLE
Process LiveBlogPackage, LiveBlogPost and ContentCollection

### PR DESCRIPTION
**Pull request by karsov**
# Description

## What

Process LiveBlogPackage, LiveBlogPost and ContentCollection

## Why

JIRA ticket UPPSF-2417

## Scope and particulars of this PR (Please tick all that apply)

- [ ] Tech hygiene (dependency updating & other tech debt)
- [ ] Bug fix
- [x] Feature
- [ ] Documentation
- [ ] Breaking change
- [ ] Minor change (e.g. fixing a typo, adding config)

___
This Pull Request follows the rules described in our [Pull Requests Guide](https://github.com/Financial-Times/upp-docs/tree/master/guides/pr-guide)

### CODE REVIEW

**Review by asparuhft — APPROVED** [review]
LGTM

File: processor/data_combiner.go [hunk 1]
```diff
@@ -69,17 +71,32 @@ func (dc DataCombiner) GetCombinedModelForAnnotations(metadata AnnotationsMessag
...
+	} else {
+		// There is nothing in the document store
+		// try to retrieve data from the content collection store
+		content, err = dc.ContentCollectionRetriever.getContent(uuid)
+		if err != nil {
+			return CombinedModel{}, err
+		}
+
+		// Content collection found but the RW-er does not include type in the content
+		if content.getUUID() != "" {
```
Comment (epavlova) [inline processor/data_combiner.go:97]: **question** : What happens if `content.getUUID() == ""`, what should be returned? `CombinedModel{}`?

Comment (karsov) [reply processor/data_combiner.go:97]: The behaviour here is not changed. If no content is found for the UUID we return the CombinedModel at the end with all fields practically empty.

However an "empty" content message is useless for the services consuming it so I'll change the logic to not forward such messages.

**Comment by karsov** [comment]
Modified the business logic so that the message is not forwarded when a content is not found on annotations publish.
Otherwise an "empty" message is sent which cannot be used anyway.

**Review by epavlova — APPROVED** [review]

**Review by asparuhft — APPROVED** [review]

**Pull request merged**